src/react-native.fs

```diff
@@ -42,7 +42,7 @@ module Program =
 
     /// Setup rendering of root ReactNative component
     let withReactNative appKey (program:Program<_,_,_,_>) =
-        AppRegistry.registerComponent(appKey, fun () -> unbox typeof<App>)
+        AppRegistry.registerComponent(appKey, fun () -> unbox JsInterop.jsConstructor<App>)
```

Comment:
Unfortunately, it's still needed because `jsConstructor` returns `obj` (maybe it should return `'b` instead) and the 2nd param of AppRegistry.registerComponent must be `unit -> ComponentClass<'a>`.